quesma/quesma/search.go

```diff
@@ -265,6 +271,10 @@ func (q *QueryRunner) executePlan(ctx context.Context, plan *model.ExecutionPlan
 }
 
 func (q *QueryRunner) handleSearchCommon(ctx context.Context, indexPattern string, body types.JSON, optAsync *AsyncQuery, queryLanguage QueryLanguage) ([]byte, error) {
+
+	decision := q.tableResolver.Resolve(table_resolver.QueryPipeline, indexPattern)
+	table_resolver.TODO("handleSearchCommon", decision)
```

Comment:
Do you already thought about dispatch implementation? (functionality of current `TODOs`)

Reply:
`TODO` is a marker function. It will be removed.

Reply:
I know it's marker but you need to do something based on decision and I'm asking about this implementation

Reply:
I had a vision, but I can't tell you how it will be implemented.